ofdb-application/src/send_update_reminders.rs

```diff
@@ -22,7 +22,7 @@ where
     let start_time = Instant::now();
     let unsent_reminders = {
         let conn = connections.shared()?;
-        ofdb_core::usecases::find_unsent_reminders(
+        usecases::find_unsent_reminders(
```

Comment:
Safe splitting into multiple transactions would require more efforts and intermediate states that are persisted in the database. But I guess it is ok here, unless we are experience frequent issues with duplicate reminders. Nevertheless, adding a comment why this decision has been made and what are the possible consequences would help.